secator/runners/_base.py

```
@@ -693,6 +783,22 @@ def _merge_opts_defaults(self):
+		while a runner temporarily disables hooks during build_celery_workflow).
+
+		Runs the resolved on_item hooks (update_finding) directly, bypassing the enable_hooks gate that
+		run_hooks enforces, so the `results` view still serves these items. No in-memory buffer. No-op
+		when no store driver is registered.
+		"""
+		if not isinstance(item, tuple(OUTPUT_TYPES)):
+			return
+		for hook in self.resolved_hooks.get('on_item', []):
+			try:
+				hook(self, item)
+			except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)